src/sacn/source_state.c

```diff
@@ -319,8 +315,12 @@ void process_universe_null_pap_transmission(SacnSource* source, SacnSourceUniver
   {
     // Send 0x00 data & reset the keep-alive timer
     send_universe_multicast(source, universe, universe->null_send_buf);
-    send_universe_unicast(source, universe, universe->null_send_buf);
-    process_null_sent(universe);
+    send_universe_unicast(source, universe, universe->null_send_buf, kSkipTerminatingUnicastDests);
```

Comment:
I think we need to rename the functions and member variables, etc, that refer to startcode 0 as null.  At first glance, universe->null_send_buf implied to me that it was a null buffer.  process_universe_null_pap_transmission is also ambiguous with that mindset.  Even something as generic as "data" or "dmx" would avoid the issue.

Reply:
I agree with Nick, "dmx" is clearer as to the purpose.

Reply:
Spoke offline with Nick and asked whether to use "dmx" or "level" and we ended up going with "level". Hopefully that will offer the same if not more clarity. Commit coming soon.

Reply:
Resolved in https://github.com/ETCLabs/sACN/pull/40/commits/f86be28fd19360833b579c79b751673ceb16b160